CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Changes
 
+#### Improvements
+
+- Updates `change` field on `LogEntry` model to be a json rather than a text field ([#407](https://github.com/jazzband/django-auditlog/pull/407))
```

Comment:
```suggestion
- feat: Change `LogEntry.change` field to `JSONField` rather than `TextField` ([#407](https://github.com/jazzband/django-auditlog/pull/407))
```